### DISCUSSION TITLE
Ask Community Admin any questions

### DISCUSSION THREAD

**Discussion by Evavic44** [CommunityPro/Practice-Contribution#15] [Q&A]
Do you have any issue with contributing to this community, are you experiencing any difficulty, do you want to speak to the community managers? 

Send them to this discussion.

**Comment by The-CodeINN** [comment]
Please drop the WhatsApp link

**Reply by Mannuel25** [reply]
> Please drop the WhatsApp link

https://chat.whatsapp.com/BMFVTksMg269GdgsviJzLm

**Reply by Evavic44** [reply]
Thanks @Mannuel25

**Reply by DanAdewole** [reply]
On the README.md file, the part where we're to include github link and profile image link, there is a typo with the word 'Replace'. 'Repalce was written instead of replace'

**Reply by Mannuel25** [reply]
> On the README.md file, the part where we're to include github link and profile image link, there is a typo with the word 'Replace'. 'Repalce was written instead of replace'

Oh, you can go ahead and make a PR to fix it. Thanks for spotting that out.

**Reply by DanAdewole** [reply]
Alright

**Comment by CodeHex083** [comment]
hello
